monitors/test/BalanceMonitor.js

```diff
@@ -310,4 +313,56 @@ contract("BalanceMonitor.js", function(accounts) {
     assert.isTrue(lastSpyLogIncludes(spy, "synthetic balance warning")); // Tx moved synthetic. should emit accordingly
     assert.equal(lastSpyLogLevel(spy), "error");
   });
+  it("Can override the collateral-threshold log level", async function() {
+    const alertOverrideConfig = { ...monitorConfig, logOverrides: { collateralThreshold: "error" } };
+    balanceMonitor = new BalanceMonitor(spyLogger, tokenBalanceClient, alertOverrideConfig, empProps);
+
+    // Lower the liquidator bot's collateral balance.
+    await collateralToken.transfer(tokenCreator, toWei("1001"), { from: liquidatorBot });
+    assert.equal((await collateralToken.balanceOf(liquidatorBot)).toString(), toBN(toWei("9999")).toString());
+
+    // Update monitors.
+    await tokenBalanceClient.update();
+    await balanceMonitor.checkBotBalances();
+
+    assert.equal(spy.callCount, 1);
+    assert.isTrue(lastSpyLogIncludes(spy, "Liquidator bot")); // name of bot from bot object
+    assert.isTrue(lastSpyLogIncludes(spy, "collateral balance warning")); // Tx moved collateral. should emit accordingly
+    assert.equal(lastSpyLogLevel(spy), "error");
+  });
+  it("Can override the ether-threshold log level", async function() {
+    const alertOverrideConfig = { ...monitorConfig, logOverrides: { ethThreshold: "error" } };
+    balanceMonitor = new BalanceMonitor(spyLogger, tokenBalanceClient, alertOverrideConfig, empProps);
+
+    // Lower the liquidator bot's ETH balance.
+    const startLiquidatorBotETH = await web3.eth.getBalance(liquidatorBot);
+    const amountToTransfer = toBN(startLiquidatorBotETH).sub(toBN(toWei("5")));
+    await web3.eth.sendTransaction({
+      from: liquidatorBot,
+      to: tokenCreator,
+      value: amountToTransfer.toString()
+    });
+    assert.isTrue(toBN(await web3.eth.getBalance(liquidatorBot)).lt(toBN(toWei("5"))));
+
+    // Update monitors.
+    await tokenBalanceClient.update();
+    await balanceMonitor.checkBotBalances();
+
+    assert.equal(spy.callCount, 1);
+    assert.isTrue(lastSpyLogIncludes(spy, "Liquidator bot")); // name of bot from bot object
+    assert.isTrue(lastSpyLogIncludes(spy, "Ether balance warning"));
+    assert.equal(lastSpyLogLevel(spy), "error");
+
+    // await syntheticToken.transfer(tokenCreator, toWei("1001"), { from: liquidatorBot });
```

Comment:
Reason for this commented out tests?

Reply:
Ah thats not commented out, that's just leftover Copy&Paste